features/networking/multus.feature

```diff
@@ -991,3 +991,39 @@ Feature: Multus-CNI related scenarios
     When I execute on the pod:
       | /usr/sbin/ip | a |
     Then the output should contain "192.168.1"
+
+  # @author weliang@redhat.com
+  # @case_id OCP-25909
+  @admin
+  Scenario: Assign static IP address using pod annotation			 
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/runtimeconfig-def-ip.yaml |
+      | n | <%= project.name %>                                                                                              |
+    Then the step should succeed
+    
+    #Clean-up required to erase above net-attach-def after testing done
+    And admin ensures "runtimeconfig-def-ip" network_attachment_definition is deleted from the "<%= project.name %>" project after scenario
```

Comment:
@weliang1 This step is not needed as project and all resources under it will be deleted automatically post scenario. This step is only needed when a net-attach-def is created under any system project like default or openshift-multus as automation doesn't touch any cleanup there. Automation just deletes the test projects and test resources so will be taken care here. Same comment for line 836 as well although it was merged initially. Thanks

Reply:
@anuragthehatter you are right, deleted clean-up steps in new commit. Will do the same for line 836 for test OCP-25676. Thanks!

Reply:
Thanks
/LGTM